src/cryptocurrency.iced

```diff
@@ -18,4 +20,15 @@ exports.Cryptocurrency = class Cryptocurrency extends Base
   _v_customize_json : (ret) ->
     ret.body.cryptocurrency = @cryptocurrency
 
+  _v_check : ({json}, cb) ->
+    esc = make_esc cb, "SubkeyBase::_v_check"
+    await super { json }, esc defer()
+    if not (section = json.body.cryptocurrency)?
+      err = new Error "needed a cryptocurrency section"
+    else
+      [err,ret] = address.check_btc_or_zcash section.address
+      if not err? and (ret.type isnt section.type)
+        err = new Error "wrote cryptocurrency type: wanted #{ret.type}"
```

Comment:
i think it's right as is.  I don't like putting user-supplied data into error strings